tendrl/commons/event.py

```diff
@@ -0,0 +1,25 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+from tendrl.commons.message import Message
+
+
+class EventFailed(Exception):
+    pass
+
+
+class Event(object):
+    def __init__(self, message):
+        self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        self._write(message)
+
+    def _write(self, message):
+        try:
+            socket_path = tendrl_ns.config.data['logging_socket_path']
+            self.sock.connect(socket_path)
```

Comment:
Is this a good idea to create/connect/delete sockets for every log message? I am just thinking in terms of volume of these operations. Or is it a good idea to think about some kind of socket client pooling?
@brainfunked @r0h4n